src/x11.h

```diff
@@ -155,7 +155,7 @@ class out_to_x_setting : public conky::simple_config_setting<bool> {
   virtual void cleanup(lua::state &l);
 
  public:
-  out_to_x_setting() : Base("out_to_x", true, false) {}
+  out_to_x_setting() : Base("out_to_x", false, false) {}
```

Comment:
Hmm, you're changing the default value from true to false? Was that intentional? I don't think we should be changing the default.

Reply:
Why do I have to `out_to_console=true` and people don't have to do `out_to_x`? @su8 made this decision so he must had a reason if he thought it was a good idea to do that (correcting bugs, removing bias, etc, making the settings more neutral).

Also, if leaving this on, I guess I have to always do `out_to_x=false` too. It seems like setting all `out_to_??` off... would lead to fewer required configs for `console`, `ncurse`, `http`, etc.

Reply:
Also, I test conky using `out_to_console` because it's easier to test variables (without x). `x` get runs in the background too due to default `out_to` behavior.